### DISCUSSION TITLE
[4.1] Article templates are linked in article

### DISCUSSION THREAD

**Discussion by N6REJ** [joomla/joomla-cms#47073] [2. Cold Features]
### Is your feature request related to a problem? Please describe.

When a article template is entered into an article using the stamp tool, its basically a simple static cut/paste.

### Describe the solution you'd like
Changes to the article template are also changed in related articles.  It's highly annoying and inefficient to have to manually reload every article that was ever created because of a simple change in the template.

**Comment by brianteeman** [comment]
Are you referring to content templates in tinymce https://youtu.be/OyZQ63Ahnt4

If so then I cannot replicate that at all. As you state its just a simple cut/paste so what you describe is impossible.

**Comment by N6REJ** [comment]
its far from impossible.  It's a feature REQUEST!  if the article template was "hotlinked" where on article save it was automatically updated in some fashion.. plugin maybe?.. then anytime they edit the article template it would update the article.
or, articled template inserts ( for example ) {my article template] then its rendered at runtime just like [fields xx} are.
or lastly a method where {field xx} is processed if its inside a text area field.
so for example
field 3 = article template text area - Paste article template in "default value"
fiield 1 = name
field 2 = age

Article template
----------
Hi {field 1} I see you are {field 2} years old

Article
-----------
{field 3}


Now the article template is completely dynamic!

**Comment by brianteeman** [comment]
Ah - it read that you were saying  a change to the template changed the existing articles.

~What you are asking for can be done. https://www.tiny.cloud/docs/plugins/opensource/template/~

It's just a copy/paste with some xtras so your _render at runtime_ would not be possible. It would also mean that the content is not searchable as it would not exist in th database.

**Comment by brianteeman** [comment]
Could someone add a new feature label please

**Comment by brianteeman** [comment]
I don't see the point in leaving feature requests open forever. Can the release team @rdeutz please add this to their decision queue.

**Comment by chmst** [comment]
Maintainers care for pull requests, not for issues. We could move this to discussions. @Hackwar your opinion?

**Comment by brianteeman** [comment]
I said Release Team not maintainers
